test/test_jit.py

```diff
@@ -2996,6 +2996,41 @@ def stuff3(x):
             return torch.ones(x), x
         self.checkScript(stuff3, ([3, 2],))
 
+    # to avoid defining sum_list in multiple tests
+    def get_sum_list_fn(self):
```

Comment:
You can put all these (`test_sum_list_diff_elms`, `test_sum_list_empty`, etc) into a single test and define `get_sum_list_fn` in there to avoid having non-test functions at the class level